docs/installation.rst

```diff
@@ -4,105 +4,226 @@
+
+      As an example of how to pass arguments, following line runs the ``dredd init`` command::
+
+         C:\Users\Susan> docker run -it -v ${pwd}:/api -w /api apiaryio/dredd dredd init
+
+
+.. _install-npm:
+
+npm
+---
+
+Dredd is a command-line application written in JavaScript, or to be more precise, in `Node.js <https://nodejs.org>`__. If Docker isn't an option for you, it's possible to install Dredd using the `npm <https://www.npmjs.com>`__ package manager. To install Dredd's dependencies, you'll also need to have `Git <https://git-scm.com/>`__ available.
+
 
 .. _install-nodejs:
 
-Install Node.js
----------------
+Installing Node.js
+~~~~~~~~~~~~~~~~~~
+
+.. tabs::
 
-macOS
-~~~~~
+   .. group-tab:: macOS
 
-1. Install Node.js.
+      - If you’re using `Homebrew <https://brew.sh/>`__, run ``brew install node``
+      - Otherwise `download Node.js <https://nodejs.org/en/download/>`__ from the official website and install it using the downloaded installer
+      - Make sure both ``node --version`` and ``npm --version`` work in your Terminal
+      - Node.js needs to be at least version 6
 
-   -  If you’re using `Homebrew <https://brew.sh/>`__, run ``brew install node``.
-   -  Otherwise `download Node.js <https://nodejs.org/en/download/>`__ from the official website and install Node.js using the downloaded installer.
+   .. group-tab:: Linux
 
-2. Make sure both ``node --version`` and ``npm --version`` work in your Terminal.
+      - `Install Node.js as a system package <https://nodejs.org/en/download/package-manager/>`__
+      - In case your Linux distribution calls the Node.js binary ``nodejs``, please `follow this advice <https://stackoverflow.com/a/18130296/325365>`__ to have it as ``node`` instead
+      - Make sure both ``node --version`` and ``npm --version`` work in your Terminal
+      - Node.js needs to be at least version 6
 
-Windows
-~~~~~~~
+   .. group-tab:: Windows
 
-1. `Download Node.js <https://nodejs.org/en/download/>`__ from the official website and install Node.js using the downloaded installer.
-2. Make sure both ``node --version`` and ``npm --version`` work in your Command Prompt.
+      - `Download Node.js <https://nodejs.org/en/download/>`__ from the official website and install it using the downloaded installer
+      - Make sure both ``node --version`` and ``npm --version`` work in your Command Prompt
+      - Node.js needs to be at least version 6
 
-Linux
-~~~~~
+.. note::
+   You can also use `nvm <https://github.com/creationix/nvm>`__ to install Node.js on your computer.
 
-1. `Install Node.js as system package <https://nodejs.org/en/download/package-manager/>`__.
-2. Make sure both ``node --version`` and ``npm --version`` work in your Terminal.
+.. warning::
+   If your internet connection is restricted (VPN, firewall, proxy), you need to run following additional commands to configure npm:
 
-Pro Tips
-~~~~~~~~
+   .. code-block:: text
```

Comment:
Might be better to link to the npm's docs on this use case as the npm commands could change https://docs.npmjs.com/misc/config#https-proxy

Reply:
I decided to keep the commands there. The official docs do not contain easy-to-copy-paste examples and people who don't know how to work with npm might get just more confused on what they're supposed to do to fix the problem. I'll link the docs tho.

Reply:
(These proxy sections are added based on real people struggling with it and reporting me back that the installation guide doesn't work for them... and they were not very skilled with npm.)